docs/Buttons/UnsubscribeButton/index.md

```diff
@@ -0,0 +1,88 @@
+## UnsubscribeButton
+
+Type
+: Function  
+
+Sender
+: SDL
+
+Purpose
+: Unsubscribe mob app from a button
```

Comment:
@KhrystynaDubovyk Could you please correct the typo "mob app"?

Reply:
@ashwink11, please see https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/253/commits/c61ac02ccc8d459f937fbc67c68cf7734ccad09f